db/migrate/20210518182324_make_notifications_polymorphic_to_subject.rb

```diff
@@ -0,0 +1,8 @@
+class MakeNotificationsPolymorphicToSubject < ActiveRecord::Migration[5.2]
+  def change
+    change_table :notifications do |t|
+      t.remove :appointment_id
+      t.references :subject, polymorphic: true, type: :uuid, null: true
```

Comment:
Does this also need a `subject_type`? I suppose this works at least until we add a second subject type.

Reply:
This adds the subject_id and subject_type automatically.

Reply:
I see. Missed that in the schema.